.github/workflows/manual_gh_release_page.yml

```diff
@@ -64,18 +64,25 @@ jobs:
     outputs:
       matrix: ${{ steps.matrix.outputs.matrix }}
     steps:
+      - uses: actions/checkout@v5.0.0
+        with:
+          submodules: true
+
       - name: Checkout flavors of the release commit
-        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # pin@v5.0.0
+        uses: actions/checkout@v5.0.0
```

Comment:
any reason to revert the commit-based versioning? we're doing this on purpose, please don't revert that.